lib/ansible/plugins/connection/ssh.py

```diff
@@ -153,7 +154,7 @@
       retries:
           # constant: ANSIBLE_SSH_RETRIES
           description: Number of attempts to connect.
-          default: 3
+          default: 0
```

Comment:
3 is the correct default

Reply:
Yeah I was trying to not change any behavior and since we used the constant and the constant defaulted to `0`, I thought it best to make this `0` too.